rest_framework-stubs/serializers.pyi

```diff
@@ -87,31 +87,28 @@ ALL_FIELDS: str = ...
 
 _MT = TypeVar("_MT", bound=Model)  # Model Type
 _IN = TypeVar("_IN")  # Instance Type
-_DT = TypeVar("_DT")  # Data Type
-_VT = TypeVar("_VT")  # Value Type
-_RP = TypeVar("_RP")  # Representation Type
 
-class BaseSerializer(Generic[_VT, _DT, _RP, _IN], Field[_VT, _DT, _RP, _IN]):
+class BaseSerializer(Generic[_IN], Field[Any, Any, Any, _IN]):
```

Comment:
If we are going with all these generic params in Field I would move _IN to first place to make it uniform with Serializer

Reply:
Its not used as often in field